src/Navbar.js

```diff
@@ -61,7 +62,9 @@ class Navbar extends Component {
 
     let navbar = (
       <nav className={navCSS}>
-        <div className="nav-wrapper">
+        <div
+          className={centerChildren ? 'nav-wrapper container' : 'nav-wrapper'}
```

Comment:
question, instead of adding a new interface prop, can't we add `container` as a className?
`<Navbar className='container' />`

Reply:
The benefit of adding an interface prop is that the users would have the choice to centered or not their Navbar child components as shown in the conversation images. If we set the class name to container, `<div className="container">` it will only show the Navbar child components centered.

I tested it by applying the recommendation with `alignLinks="left"` and the #981 fix, which 
rendered as:
<img width="839" alt="actual" src="https://user-images.githubusercontent.com/8462791/72661391-451be880-39a7-11ea-8fe4-08836d1816a9.png">



 rather than:
<img width="1680" alt="expected" src="https://user-images.githubusercontent.com/8462791/72661371-0be37880-39a7-11ea-9a8e-f4f030d8f7d7.png">